core/shared/src/main/scala/org/http4s/websocket/WebSocketContext.scala

```diff
@@ -22,20 +22,45 @@ import cats.syntax.all._
 import cats.~>
 import org.typelevel.ci.CIStringSyntax
 
+import scala.concurrent.duration.FiniteDuration
+
 final case class WebSocketContext[F[_]](
     webSocket: WebSocket[F],
     headers: Headers,
     failureResponse: F[Response[F]],
-) {
+    autoPing: Option[(FiniteDuration, WebSocketFrame.Ping)],
+) { self =>
+  // required for binary compatibility
+  def this(webSocket: WebSocket[F], headers: Headers, failureResponse: F[Response[F]]) =
+    this(webSocket, headers, failureResponse, None)
+
+  // required for binary compatibility
+  def copy(
+      webSocket: WebSocket[F] = self.webSocket,
+      headers: Headers = self.headers,
+      failureResponse: F[Response[F]] = self.failureResponse,
```

Comment:
remove the defaults

Reply:
Done, but I am still stuck with mima. Need some rope here :/

Reply:
@armanbilge After removingthe defaults I fail to get over the mima error message
```
synthetic method copy$default$1()org.http4s.websocket.WebSocket in class org.http4s.websocket.WebSocketContext does not have a correspondent in current version
```